.github/workflows/main.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         run: |
           cd espresso-jshell
           ./build-espresso-jshell.sh
-          echo "Math.PI" | ./espresso-jshell
+          echo "Math.PI" | ./espresso-jshell -Dorg.graalvm.home=$GRAALVM_HOME
```

Comment:
Why does `$GRAALVM_HOME` need to be passed in? Is there no way to detect this from inside the `espresso-jshell`?

Reply:
Fixed! Now it tries to set/initialize `org.graalvm.home` and `java.home` from the `GRAALVM_HOME` env. variable.
Running just `./espresso-jshell` may give the false illusion that the executable is fully standalone.
I added a small section to the README explaining this.